### ISSUE TITLE
Find a way to provide default button configuration

### ISSUE THREAD

**Issue by spbnick** [DIGImend/digimend-kernel-drivers#117]
Huion requested to have default hotkey configuration for buttons on tablet frames. We need to figure out how to do that.

**Comment by spbnick** [comment]
Yes, libwacom is probably where we'll have to go, and also submitting descriptions of Huion tablets there.

**Comment by reya276** [comment]
I'm actually trying to develop a config tool for each tablet listed within this project.

![designtablets](https://user-images.githubusercontent.com/25517972/38894024-1041cd5a-425a-11e8-9d37-cea9d4e0115d.png)

**Comment by ozeszty** [comment]
Nice, are you going to support multiple (e.g. per application or task) hotkeys' configs? Even in one app, multiple configs could be useful (painting or animating in Krita).

Some ideas for development:

- Some pens have rubber on the other end serving as another button.
- It should be possible to emulate another button when both buttons are pressed together (or in minimal interval, e.g. < 300 ms), like with touchpads and getting middle-click by pressing left and right buttons together.

**Comment by megamuteki** [comment]
1.I would like to translate this code into japanese.
2.I can provide photoes and default key-code.
3.I have many XP-PEN tablets, if you need and I have one, I can dump data.

**Comment by Lvaskz** [comment]
Hi, this is very usefull. Do you have any advance about it?

**Comment by reya276** [comment]
@Lvaskz Not yet, trying to see if I can get some developers to help me out.

**Comment by reya276** [comment]
@megamuteki I sent you some emails about the project, send me anything you have.

**Comment by reya276** [comment]
@Lvaskz no not really I've been swamped at work so I have not had time to get back to it. I'm waiting on some other folks to help me with the scripts side of it.

**Comment by Lvaskz** [comment]
I see, thanks to reply. Well I will wait.

**Comment by megamuteki** [comment]
I have been tried to work on Ubntu 16.04 LTS.
The scripts does not work on ubuntu 16.04 LTS easily  becuase of glib issues.
I reccomend to use higher than Ubuntu 18.04.

**Comment by megamuteki** [comment]
I have retrieved XP-PEN tablet datas.
If  you need more info, please let me know.
I have many xp-pen tablet.
[XpPenTabletData.zip](https://github.com/DIGImend/digimend-kernel-drivers/files/2161580/XpPenTabletData.zip)

**Comment by ozeszty** [comment]
https://github.com/KDE/wacomtablet rocks! On top of what GNOME's tablet settings provide, it supports,  among other features, config profiles. It played well with Huion H610 Pro.

**Comment by Lvaskz** [comment]
hi, a long time has passed.!!! and I wonder if it has some progress?

**Comment by Lvaskz** [comment]
Hi, any progress? Or definitively this willl not happen?
